apps/prairielearn/package.json

```diff
@@ -120,7 +120,7 @@
     "klaw": "^4.1.0",
     "lodash": "^4.17.21",
     "loopbench": "^2.0.0",
-    "lru-cache": "^10.4.0",
+    "lru-cache": "^11.0.0",
```

Comment:
Only breaking change is the drop of support for Node <20, so this should be safe.